src/type.zig

```diff
@@ -1596,6 +1640,123 @@ pub const Type = extern union {
         };
     }
 
+    /// Asserts the type is error_set or error_set_single or anyerror and that if it is error_set, it's decl has been analyzed
+    /// If it is error_set_single, it will return the []const u8, otherwise a pointer to the map with the error_set or void for anyerror
+    pub fn getErrs(self: Type) union(enum) { err_single: []const u8, multiple: *std.StringHashMapUnmanaged(u16), anyerror: void } {
```

Comment:
I don't think putting this in type.zig is viable - we need the concept of "resolving" an error set which is the part where it figures out what all errors are in the set. Resolving an error set could cause compile errors, which will need to be attached to a Module. So I think this code does not belong here.